students/HarveyHerela/lesson02/part1.py

```diff
@@ -0,0 +1,10 @@
+horizontalEdge = "+ - - - - + - - - - +"

```

Comment:
Python naming convention is to use snake case (horizontal_edge) for all variable and function names